### PR TITLE
Aibolit WP: split long sentence. 

### LINKED ISSUE #603: Aibolit WP: write it better
https://github.com/cqfn/aibolit/blob/7ba9e693d551f110c92521af02b5fa09f33163bc/wp/sections/usage_scenarios.tex#L3

write this sentence better

### PR DESCRIPTION
**Pull request by acheshkov**
closes #603 
the long sentence is split and written simpler.

### CODE REVIEW

File: wp/sections/usage_scenarios.tex [hunk 1]
```diff
@@ -1,6 +1,7 @@
 \subsection{Aibolit Index}
 
-In addition to the recommendation functionality, we designed a feature to measure the overall quality of developer's source code, \textit{Aibolit Index}.
+In addition to the recommendation functionality, we designed an \textit{"Aibolit Index"}.
```
Comment (KatGarmash) [inline wp/sections/usage_scenarios.tex:3]: I actually don't think you should split this sentence.

I'd just rewrite a bit: 
In addition to the recommendation functionality, we prppose \textit{Aibolit Index}, a score that measures the overall quality of developer's source code.

Comment (acheshkov) [reply wp/sections/usage_scenarios.tex:3]: I'm okay even with the previous version of the sentence, except this part `of developer's source code`. I suggest this refactoring, as I understand the professor's comments.

Comment (KatGarmash) [reply wp/sections/usage_scenarios.tex:3]: Ah then maybe:

"""
In addition to the recommendation functionality, we proppose \textit{Aibolit Index}, a score that measures the overall quality of a given software project.
"""

I think "a given" sounds better than "some" in this context.

Comment (acheshkov) [reply wp/sections/usage_scenarios.tex]: I like it

**Pull request merged**